### PR TITLE
Improve request to join from logged out/unregistered flow

### PR DESCRIPTION
**Pull request by Connoropolous**
**How it looks now**
only when the url provides an indication that they arrived here via clicking 'request to join' community while unauthed.

screen shot 2017-01-13 at 4 22 43 pm

screen shot 2017-01-13 at 4 22 32 pm

screen shot 2017-01-13 at 4 26 03 pm

### CODE REVIEW

**Comment by levity** [comment]
If they got to the signup page from a request-to-join button, the join request will be sent automatically once they finish signup and get redirected. So we wouldn't want to tell them to press the button again. Probably the best thing to do here is to check the value of `next` in the query params, and if it contains `join=true`, omit the subtitle entirely.

**Comment by levity** [comment]
Thanks for being proactive about creating a PR in response to user feedback though! 👍

**Comment by Connoropolous** [comment]
"Probably the best thing to do here is to check the value of next in the query params, and if it contains join=true, omit the subtitle entirely." 👍  will modify this to do that. I didn't know that was the behaviour

**Comment by Connoropolous** [comment]
It previously looked like this, in terms of confirmation, look to the bottom left of the image

screen shot 2017-01-13 at 3 02 51 pm

File: src/containers/Signup.js [hunk 1]
```diff
@@ -88,15 +88,17 @@ export default class Signup extends React.Component {
...
-    const subtitle = community ? null
-      : `If you're trying to join an existing community, please use the special
-        invitation link that you received from your community manager.`
-    const { email } = query
-
+    const { email, next } = query
+    const willCreateJoinRequest = next && 
+      (next.indexOf('join=true') > -1 || next.indexOf('join%3Dtrue') > -1)
+    let subtitle = willCreateJoinRequest ? 'Once you sign up, a request to join the community will be sent.' : ''
+    subtitle = !community && !willCreateJoinRequest ? `If you're trying to join an existing community, please use the special
```
Comment (levity) [inline src/containers/Signup.js]: This would read better as an if-clause

Comment (Connoropolous) [reply src/containers/Signup.js]: true true. will fix that up

**Comment by levity** [comment]
Feel free to merge after making that change.

**Pull request merged**